src/utils.ts

```diff
@@ -6,3 +6,41 @@ export function getElementById<T extends HTMLElement>(id: string): T {
     return ElementId as T;
   }
 }
+
+export class ToastManager {
+  constructor() {}
+
+  showSuccessMessage(message: string, options: number): void {
```

Comment:
options must be object where there is property `delay`. Use interface or word `type`  in typescript for options type.